instruments/Reverb1.orc

```diff
@@ -22,17 +22,33 @@ instr Reverb1
 endin
 
 instr Reverb1Wet
+  if (p3 == 0) then
```

Comment:
I should remove these checks. I'm pretty sure they are useless because an orchestra event can't initialize when p3 (duration) equals 0.